.github/workflows/windows_builds.yml

```
@@ -49,6 +49,17 @@ jobs:
       - name: Setup python and scons
         uses: ./.github/actions/godot-deps
 
+      - name: Download pre-built ANGLE static libraries
+        uses: dsaltares/fetch-gh-release-asset@1.1.1
```

```suggestion
        uses: dsaltares/fetch-gh-release-asset@1.1.2
```
Or `@1` might be enough to use the latest?